3_javascript/javascript1/week3/script3.js

```diff
@@ -11,26 +11,23 @@ const names = [
   ];
   const nameToRemove = "Ahmad";
   
-let remainingNames = names.filter(name => name !==nameToRemove);  
+const remainingNames = names.filter(name => name !==nameToRemove);  
   console.log(remainingNames); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
 
 
 //   When will we be there??
-// Write a function where you speficy your speed in km/h and how far you have to go in km. 
-// The function has to return the time it will take to arrive at your destination. 
-// The time should be formatted like this: 
-// 3 hours and 34 minutes.
 
-function totalTravelTime (travelInformation) {
+
+function calculateTravelDuration (travelInformation) {
   const {speed, destinationDistance} = travelInformation;
   const totalHours = destinationDistance / speed;
   const hours = Math.floor(totalHours);
   const minutes = Math.round((totalHours - hours) * 60);
 
   let calculatedTime = `${hours} hours`;
 
-  if (minutes > 0) {
-    calculatedTime += ` and ${minutes} minutes`;
+  if (minute > 0) {
+    calculatedTime += ` and ${minute} minutes`;
```

Comment:
My previous comment was about how would it display when **minute = 1** - `1 minutes`? Or maybe should be `1 minute`.